data_broker.js

```diff
@@ -40,6 +41,9 @@ class DataBroker extends EventTarget {
   }
 
   set alignmentUrl(_) {
+    if (this._alignmentUrl !== _) {
```

Comment:
You're detecting the alignmentUrl change in two places. You should use the one below. Having this here will cause the event to be emitted even when _tryUpdate doesn't trigger.